ios/ReactNativePageView.m

```diff
@@ -384,22 +384,28 @@ - (UIViewController *)createChildViewController:(UIView *)view {
 }
 
 - (void)goTo:(NSNumber *)index animated:(BOOL)animated {
-    if (_currentIndex >= 0 &&
-        index.integerValue < _childrenViewControllers.count) {
-        
-        _reactPageIndicatorView.currentPage = index.integerValue;
+    if (_currentIndex >= 0) {
         UIPageViewControllerNavigationDirection direction =
         (index.integerValue > _currentIndex)
         ? UIPageViewControllerNavigationDirectionForward
         : UIPageViewControllerNavigationDirectionReverse;
         
-        UIViewController *viewController =
-        [_childrenViewControllers objectAtIndex:index.integerValue];
-        [self setReactViewControllers:index.integerValue
-                                 with:viewController
+        
+        NSInteger indexToDisplay = index.integerValue;
+        UIViewController *controlerToDisplay = nil;
```

Comment:
nit, `UIViewController *controlerToDisplay` should be enough

Reply:
fixed